app/views/alchemy/admin/pictures/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="zoomed-picture-background">
-  <%= image_tag @picture.url(format: @picture.image_file_format) || "alchemy/missing-image.svg" %>
+  <%= image_tag @picture.url || "alchemy/missing-image.svg" %>
```

Comment:
I believe this is an easy backport, let's get it into 5.0

Reply:
https://github.com/AlchemyCMS/alchemy_cms/pull/1902